libvirt/tests/src/migration/migration_with_disk/migration_with_fdgroup.py

```
@@ -0,0 +1,192 @@
+
+
+def verify_fd_open(file_paths, test):
+    """
+    Verify that virsh and virtqemud processes have the FDs open
+
+    :param file_paths: list of file paths to check
+    :param test: test object
+    """
+    for file_path in file_paths:
+        cmd = f"lsof {file_path} | grep -E 'qemu' || true"
+        result = process.run(cmd, ignore_status=True, shell=True)
```

Ruff (0.15.6)

[error] Function call with `shell=True` parameter identified, security issue

(S604)